src/Platform/Microsoft.Testing.Platform/Requests/NopFilter.cs

```diff
@@ -1,6 +1,10 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.Diagnostics.CodeAnalysis;
+
 namespace Microsoft.Testing.Platform.Requests;
 
-internal sealed class NopFilter : ITestExecutionFilter;
+[Experimental("TAEXP001")]
+[SuppressMessage("ApiDesign", "RS0016:Add public types and members to the declared API", Justification = "Experimental API")]
```

Comment:
I can't make my mind on whether it's bette rto have this part of the public API file or not.

Reply:
We will keep the suppression for experimental public APIs.